aeron-driver/src/main/c/aeron_driver_context.h

```diff
@@ -125,7 +125,7 @@ typedef struct aeron_driver_context_stct
     sm_receiver_tag;
     struct
     {
-        int32_t receiver_group_min_size;                     /* aeron.flow.control.receiver.group.min.size = 0 */
+        uint32_t receiver_group_min_size;                    /* aeron.flow.control.receiver.group.min.size = 0 */
```

Comment:
Why change the type to be inconsistent with Java types? Elsewhere we try to remain consistent.

Reply:
Mostly because it removes a number of casts that changed signedness at point of use.

Reply:
Changed it back to int32_t